source/extensions/tracers/opentelemetry/samplers/dynatrace/sampler_config_provider.cc

```diff
@@ -0,0 +1,21 @@
+#include "source/extensions/tracers/opentelemetry/samplers/dynatrace/sampler_config_provider.h"
+
+#include "source/common/common/enum_to_int.h"
+#include "source/common/http/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+SamplerConfigProviderImpl::SamplerConfigProviderImpl(
+    Server::Configuration::TracerFactoryContext& /*context*/,
+    const envoy::extensions::tracers::opentelemetry::samplers::v3::DynatraceSamplerConfig& config)
+    : sampler_config_(config.root_spans_per_minute()) {}
```

Comment:
Our goal is that sampling configuration is obtained in the background from the Dynatrace API, to achieve dynamic sampling. 

For this PR we went with this simple approach of returning the value from the configuration. The fetching from the API will be added in a follow up PR.